.github/workflows/tag.yml

```diff
@@ -1,5 +1,5 @@
 name: Tag
-on: workflow_dispatch
+on: workflow_call
```

Comment:
I think, to avoid triggering a change that causes the build to run again, you need to add the modified path (`settings.gradle`) to the `paths-ignore: [ '*.md' ]` block in `build-and-test.yml`. See https://github.com/DataBiosphere/java-pfb/blob/sh-aj-1095-publilsh/.github/workflows/build-and-test.yml#L6